sdk/include/psdk/tcpestats.h

```diff
@@ -0,0 +1,216 @@
+    ULONG AbruptTimeouts;
+    ULONG PktsRetrans;
+    ULONG BytesRetrans;
+    ULONG DupAcksIn;
+    ULONG SacksRcvd;
+    ULONG SackBlocksRcvd;
+    ULONG CongSignals;
+    ULONG PreCongSumCwnd;
+    ULONG PreCongSumRtt;
+    ULONG PostCongSumRtt;
+    ULONG PostCongCountRtt;
+    ULONG EcnSignals;
+    ULONG EceRcvd;
+    ULONG SendStall;
+    ULONG QuenchRcvd;
+    ULONG RetranThresh;
+    ULONG SndDupAckEpisodes;
+    ULONG SumBytesReordered;
+    ULONG NonRecovDa;
+    ULONG NonRecovDaEpisodes;
+    ULONG AckAfterFr;
+    ULONG DsackDups;
+    ULONG SampleRtt;
+    ULONG SmoothedRtt;
+    ULONG RttVar;
+    ULONG MaxRtt;
+    ULONG MinRtt;
+    ULONG SumRtt;
+    ULONG CountRtt;
+    ULONG CurRto;
+    ULONG MaxRto;
+    ULONG MinRto;
+    ULONG CurMss;
+    ULONG MaxMss;
+    ULONG MinMss;
+    ULONG SpuriousRtoDetections;
+} TCP_ESTATS_PATH_ROD_v0, *PTCP_ESTATS_PATH_ROD_v0;
+
+typedef struct _TCP_ESTATS_PATH_RW_v0 {
+    BOOLEAN EnableCollection;
+} TCP_ESTATS_PATH_RW_v0, *PTCP_ESTATS_PATH_RW_v0;
+
+typedef struct _TCP_ESTATS_REC_ROD_v0 {
+    ULONG  CurRwinSent;
+    ULONG  MaxRwinSent;
+    ULONG  MinRwinSent;
+    ULONG  LimRwin;
+    ULONG  DupAckEpisodes;
+    ULONG  DupAcksOut;
+    ULONG  CeRcvd;
+    ULONG  EcnSent;
+    ULONG  EcnNoncesRcvd;
+    ULONG  CurReasmQueue;
+    ULONG  MaxReasmQueue;
+    SIZE_T CurAppRQueue;
+    SIZE_T MaxAppRQueue;
+    UCHAR  WinScaleSent;
+} TCP_ESTATS_REC_ROD_v0, *PTCP_ESTATS_REC_ROD_v0;
+
+typedef struct _TCP_ESTATS_REC_RW_v0 {
+    BOOLEAN EnableCollection;
+} TCP_ESTATS_REC_RW_v0, *PTCP_ESTATS_REC_RW_v0;
+
+typedef struct _TCP_ESTATS_SEND_BUFF_ROD_v0 {
+    SIZE_T CurRetxQueue;
+    SIZE_T MaxRetxQueue;
+    SIZE_T CurAppWQueue;
+    SIZE_T MaxAppWQueue;
+} TCP_ESTATS_SEND_BUFF_ROD_v0, *PTCP_ESTATS_SEND_BUFF_ROD_v0;
+
+typedef struct _TCP_ESTATS_SEND_BUFF_RW_v0 {
+    BOOLEAN EnableCollection;
+} TCP_ESTATS_SEND_BUFF_RW_v0, *PTCP_ESTATS_SEND_BUFF_RW_v0;
+
+typedef struct _TCP_ESTATS_SND_CONG_ROD_v0 {
+    ULONG  SndLimTransRwin;
+    ULONG  SndLimTimeRwin;
+    SIZE_T SndLimBytesRwin;
+    ULONG  SndLimTransCwnd;
+    ULONG  SndLimTimeCwnd;
+    SIZE_T SndLimBytesCwnd;
+    ULONG  SndLimTransSnd;
+    ULONG  SndLimTimeSnd;
+    SIZE_T SndLimBytesSnd;
+    ULONG  SlowStart;
+    ULONG  CongAvoid;
+    ULONG  OtherReductions;
+    ULONG  CurCwnd;
+    ULONG  MaxSsCwnd;
+    ULONG  MaxCaCwnd;
+    ULONG  CurSsthresh;
+    ULONG  MaxSsthresh;
+    ULONG  MinSsthresh;
+} TCP_ESTATS_SND_CONG_ROD_v0, *PTCP_ESTATS_SND_CONG_ROD_v0;
+
+typedef struct _TCP_ESTATS_SND_CONG_ROS_v0 {
+    ULONG LimCwnd;
+} TCP_ESTATS_SND_CONG_ROS_v0, *PTCP_ESTATS_SND_CONG_ROS_v0;
+
+typedef struct _TCP_ESTATS_SND_CONG_RW_v0 {
+    BOOLEAN EnableCollection;
+} TCP_ESTATS_SND_CONG_RW_v0, *PTCP_ESTATS_SND_CONG_RW_v0;
+
+typedef struct _TCP_ESTATS_SYN_OPTS_ROS_v0 {
+    BOOLEAN ActiveOpen;
+    ULONG   MssRcvd;
+    ULONG   MssSent;
+} TCP_ESTATS_SYN_OPTS_ROS_v0, *PTCP_ESTATS_SYN_OPTS_ROS_v0;
+
+#endif // _TCPESTATS_
```

Comment:
```suggestion
#endif // _TCPESTATS_

```
(missing empty trailing newline)